npc_simulator/src/node.cpp

```diff
@@ -277,22 +288,22 @@ bool NPCSimulator::checkValidLaneChange(
 
   if (!change_lane) {
     RCLCPP_WARN_STREAM(
-      logger_, "no right lane, current lane id=" << current_lane_id
-                                                      << ",lane change num= " << lane_change_dir);
+      logger_, "no right lane, current lane id=" << current_lane_id <<
+        ",lane change num= " << lane_change_dir);
     return false;
   }
 
   RCLCPP_INFO_STREAM(
-    logger_, "start lane change , current_lane_id =" << current_lane_id << ", target lane_id="
-                                                          << change_lane->id());
+    logger_, "start lane change , current_lane_id =" << current_lane_id << ", target lane_id=" <<
+      change_lane->id());
   result_lane_id = change_lane->id();
   return true;
 }
 
 bool NPCSimulator::checkValidUTurn(
   const geometry_msgs::msg::Pose & obj_pose, const int current_lane_id, int & result_lane_id)
 {
-  // TODO: Is there API to get opposite adjacent lane???
+  // TODO(Tomoya Kimura): Is there API to get opposite adjacent lane???
   // get opposite pose
   geometry_msgs::msg::Pose * turn_pose = new geometry_msgs::msg::Pose(obj_pose);
```

Comment:
clang-tidy pointed out this could be a memory leak. Can this be made a stack object?

Reply:
Made into a stack object.